src/resources/js/v2/tickets-utils.js

```diff
@@ -225,4 +225,26 @@ tribe.tickets.utils = {};
 		// Return the post id for the first ticket block.
 		return $ticketsBlock.getAttribute( 'data-post-id' ) || false;
 	};
+
+	/**
+	 * Get the price of the ticket from the ticket item element
+	 *
+	 * @since TBD
+	 *
+	 * @return {float|int} The ticket price.
+	 */
+	obj.getPrice = function( $ticketItem, provider ) {
+		if ( $ticketItem ) {
+			const realPrice = $ticketItem.data( 'ticket-price' );
+			const formattedPrice = $ticketItem
+				.find( '.tribe-tickets__tickets-sale-price .tribe-amount' )
+				.text();
+			const priceString = isNaN( realPrice ) 
+				? obj.cleanNumber( formattedPrice, provider ) 
+				: realPrice;
+				return parseFloat( priceString );
+		}
+		return 0;
```

Comment:
We normally bail early under some conditions and then execute the code. I'd invert the conditional and return 0, and then execute the rest.